src/sentry/grouping/strategies/newstyle.py

```diff
@@ -719,7 +722,7 @@ def chained_exception(
         rv[name] = GroupingComponent(
             id="chained-exception",
             values=component_list,
-            tree_label=calculate_tree_label(reversed(component_list)),
+            tree_label=calculate_tree_label(list(reversed(component_list))),
```

Comment:
Otherwise I get this error:
```
src/sentry/grouping/strategies/newstyle.py:725: error: Argument 1 to "calculate_tree_label" has incompatible type "reversed[GroupingComponent]"; expected "Sequence[str | GroupingComponent]"  [arg-type]
Found 1 error in 1 file (checked 1 source file)
```

Reply:
could probably change the type from `Sequence` -> `Iterable`